charts/catalog-server/values.yaml

```diff
@@ -122,6 +122,45 @@ readinessProbe:
   # -- Failure threshold for readinessProbe
   failureThreshold: 3
 
+# -- Network policy defines who can access this application and who this applications has access to
+# @default -- check `values.yaml`
+networkpolicy:
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 0.0.0.0/0
+        except:
+        - 10.0.0.0/8
+        - 192.168.0.0/16
+        - 172.16.0.0/20
```

Comment:
why are we blocking these private/local IPs?

Reply:
Should we also accept incoming requests from management portal here? Since it needs the catalog server for source types?

Reply:
Yes! Good catch! Thank you!